docs/Basic Concepts.md

```diff
@@ -0,0 +1,61 @@
+# Cloudflow - Basic Concepts
+
+This document gives you an overview of the main building blocks of a cloudflow application.
+
+## Application
+
+A Cloudflow application is a self-contained distributed system of data processing services connected together by data streams. A Cloudflow cluster can host multiple applications.
+
```

Comment:
Is `Cloudflow cluster` the term we want to use or its more about Cloudflow runtime?

Reply:
maybe s/`self-contained distributed system of data processing services connected together by data streams`/`a collection of data processing components connected via data streams"`

Reply:
done ..